### PR TITLE
Errors in the prefetcher can be hidden by subsequent errors in flushing.

### PR DESCRIPTION
**Pull request by rt121212121**

### CODE REVIEW

File: electrumx/server/block_processor.py [hunk 1]
```diff
@@ -675,6 +675,10 @@ def spend_utxo(self, tx_hash, tx_idx):
             async with TaskGroup() as group:
                 await group.spawn(self.prefetcher.main_loop(self.height))
                 await group.spawn(self._process_prefetched_blocks())
+        except Exception:
+            # If there is an error in the prefetcher chances are the flush can error too, hiding
+            # this error.
+            self.logger.exception('error in prefetcher')
```
Comment (SomberNight) [inline electrumx/server/block_processor.py]: shouldn't you re-raise?

**Comment by rt121212121** [comment]
Ah, it was late, very good point.

**Comment by rt121212121** [comment]
I think this possibly needs to filter out logging for `CancelledError`.

**Comment by kyuupichan** [comment]
In git HEAD I'm only flushing for CancelledError as others may be a consequence, or cause, of corrupted in-memory data which we don't want to flush and which tends to trigger the assertions in the flush code

**Pull request closed without merging**